datm/atm_comp_nuopc.F90

```diff
@@ -92,8 +92,6 @@ module atm_comp_nuopc
   logical                      :: flds_presaero = .false.             ! true => send valid prescribe aero fields to mediator
   logical                      :: flds_co2 = .false.                  ! true => send prescribed co2 to mediator
   logical                      :: flds_wiso = .false.                 ! true => send water isotopes to mediator
-  character(CL)                :: bias_correct = nullstr              ! send bias correction fields to coupler (not used here)
-  character(CL)                :: anomaly_forcing(8) = nullstr        ! send anomaly forcing fields to coupler (not used here)
```

Comment:
These are needed for these features that have not been implemented yet - please put these back in.